.idea/gradle.xml

```diff
@@ -3,8 +3,9 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleHome" value="C:\Program Files (x86)\Android\Android Studio\gradle\gradle-2.2.1" />
```

Comment:
Nein. Überhaupt sind mehr Files als nötig im repo. 
Siehe hier: http://stackoverflow.com/questions/16736856/what-should-be-in-my-gitignore-for-an-android-studio-project